core/src/main/java/discord4j/core/object/entity/channel/VoiceChannel.java

```diff
@@ -171,6 +174,18 @@ public int getUserLimit() {
                 .any(memberId::equals);
     }
 
+    /**
+     * Requests to retrieve the voice connection that the client is propagating in this voice channel.
+     *
+     * @return A {@link Mono} that emits the voice connection associated with this voice channel, if present and if
```

Comment:
And here, further specifying that it might be empty if the voice connection is associated to a different channel in this guild.